.changesets/fix_lrlna_api_schema_metrics_4406.md

```diff
@@ -0,0 +1,6 @@
+
+### Trace api schema generation errors under apollo.router.operations ([Issue #4406](https://github.com/apollographql/router/issues/4406))
```

Comment:
do we really need a public facing changeset here? IMO it may be useful for users to know that a new toAPISchema implementation is being tested but not so much how its reported to us.

Reply:
i think this changeset is likely to get dropped in the release? or do we just not write one in that case?

Reply:
I tend to not write changesets in this situation, and explain why i didnt write any in the PR body